beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -2823,26 +2818,80 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         // Increment the Prometheus counter for block processing requests.
         metrics::inc_counter(&metrics::BLOCK_PROCESSING_REQUESTS);
 
+        let block_slot = unverified_block.block().slot();
```

Comment:
The diff is increased big here, but the intents to remain functionally the same. the diff with unstable ends up much smaller

Reply:
Nice!